lib/db/mysql/patches/patch-005-006.sql

```diff
@@ -14,7 +14,7 @@ CREATE TABLE refreshTokens (
 
 -- Add expiresAt column for access tokens
 
-ALTER TABLE tokens ADD COLUMN expiresAt TIMESTAMP NOT NULL;
+ALTER TABLE tokens ADD COLUMN expiresAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
```

Comment:
I have looked at the specifics here, but production runs 5.6 (and has always run that). Just to be clear on that point.